cmd/kube-proxy/app/server.go

```diff
@@ -584,15 +584,15 @@ func serveHealthz(hz healthcheck.ProxierHealthUpdater, errCh chan error) {
 	fn := func() {
 		err := hz.Run()
 		if err != nil {
-			klog.Errorf("healthz server failed: %v", err)
+			klog.ErrorS(err, "Healthz server failed")
 			if errCh != nil {
 				errCh <- fmt.Errorf("healthz server failed: %v", err)
 				// if in hardfail mode, never retry again
 				blockCh := make(chan error)
 				<-blockCh
 			}
 		} else {
-			klog.Errorf("healthz server returned without error")
+			klog.ErrorS(nil, "Healthz server returned without error")
```

Comment:
@thockin is this really an error or an Info?

Reply:
Should I change it to an Info? or waiting  someone to confirm?

Reply:
we need to wait for an approver, so if is not him, other will find the question in the review and answer it

Reply:
ok, thanks a lot

Reply:
What is shown here is Info, If want error, Maybe migrate as
klog.ErrorS(fmt.Errorf("Healthz server returned without error"), "")
/cc @serathius, In this case, do we migrate according to info or err?

Reply:
I think Error is appropriuate

Reply:
why a "no error" message is treated as error, is just for learning purposes, is not that we use stderr here or something